config/locales/client.en.yml

```diff
@@ -1131,6 +1131,7 @@ en:
         mute_option_title: "You will not receive any notifications related to this user."
         normal_option: "Normal"
         normal_option_title: "You will be notified if this user replies to you, quotes you, or mentions you."
+        silence_close: "Silence close notifications"
```

Comment:
Let's invert this a bit to fit in preferences/tracking . So it would be "Consider topics unread when they are closed", and default on.

Reply:
kk sounds good. Wow bold with the default on, I like it.